src/System.Device.Gpio/System/Device/Spi/SpiConnectionSettings.cs

```diff
@@ -49,5 +51,13 @@ public SpiConnectionSettings(int busId, int chipSelectLine)
         /// The frequency in which the data will be transfered.
         /// </summary>
         public int ClockFrequency { get; set; }
+        /// <summary>
+        /// Specifies which bits will be transfered first over the wire.
+        /// </summary>
+        public DataFlow DataFlow { get; set; } = DataFlow.MsbFirst;
```

Comment:
NIT: I prefer more like "... transferred first on the SPI bus.".  We use similar wording in other SPI/I2C files.